SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -363,9 +360,9 @@ FORCEINLINE Worker_ComponentId GetClientAuthorityComponent(bool bUsingRingBuffer
 	return bUsingRingBuffers ? CLIENT_ENDPOINT_COMPONENT_ID : CLIENT_RPC_ENDPOINT_COMPONENT_ID_LEGACY;
 }
 
-FORCEINLINE Worker_ComponentId GetCrossServerRPCComponent(bool bUsingRingBuffers)
+FORCEINLINE Worker_ComponentId GetCrossServerRPCComponent()
```

Comment:
Nit - why forceinline. Presumably just to fit in with the rest of stuff. But it seems odd to use it here.

Reply:
Actually if we are unifying them then do we still need this function?

Reply:
forceinline just because it already was, and might as well be as I can't think of a reason why not

why we still have this function is a better question- I left it even though right now it's a single choice, it will be a centralised place to change if we do add more cross server rpc methods with different components

Reply:
In which case it would need to be a more complicated function though. I suspect that generally the RPC service will know what kind of RPC it is sending, but if it doesn't it will be dependent on a fair amount of data that would need to be passed to the function.

Reply:
However if you want to keep the function then it doesn't do much harm and might be useful to already be using a function.

Reply:
For why not force inline I'd say it's mostly a matter of hygiene. Having compiler optimisation flags in everywhere when they almost certainly aren't needed seems bad. I'd rather put them in when we think they are needed rather than just use them because we can. Not everything is better when inlined and this is the kind of thing the compiler doesn't need our help with inlining.

Reply:
removed the force inlines in this class, fully on board with not adding those without evidence (seems like premature optimization) and removed the function and used the constant directly.